index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="index.css">
+  <title>Music Conference</title>
+</head>
+
+<body>
+  <header>
+    <div class="header1">
+      <ul class="header-links">
+        <li><i class="fa-brands fa-facebook-f"></i></li>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li>English</li>
+        <li>my page</li>
+        <li>log in</li>
+      </ul>
+    </div>
+    <nav>
+      <div class="logo-container"><a href="index.html"><img src="assets/images/logo.png" id="logo" alt="logo image"></a>
+      </div>
+      <img src="assets/images/menu-icon.svg" id="menu-icon" alt="hamburger menu icon" class="icon">
+      <ul class="nav-links" id="nav-links">
+        <img src="assets/images/menu-close-Icon.png" id="menu-close-icon" class="menu-icon" alt="close menu icon">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#">Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main id="main">
+    <section class="hero">
+      <div class="hero-text-content">
+        <span class="sub-title">Hello! Music Producers!</span>
+        <h1 class="title" id="title">Tritone Music Producers conference 2023</h1>
```

Comment:
- [ ] You have done a great job increasing the size of the header as requested of you by the previous reviewer. Kindly position it so that it fits properly as it is provided in the Behance design.
- [ ] Also, as requested, kindly ensure that you use the same color for the texts as it is provided in the Behance design

**This is how your design looks like now:**

![image](https://user-images.githubusercontent.com/86318284/212381420-16e5e07b-73fe-4790-8108-5542c35f8f33.png)


**This is the desired design**

![image](https://user-images.githubusercontent.com/86318284/212381510-7f422515-6f93-49e2-8af2-72aa84a69f25.png)

Reply:
thanks for pointing that out!
fixed!